components/SocShare.tsx

```diff
@@ -18,6 +20,47 @@ export default function SocShare({
 
   const { copyLink, isCopiedTooltip } = useCopyToClipboard()
 
+if(color === undefined){
+return (
+    <ul
+      className={`${
+        isRoundedIcons
+          ? 'social-icon icon-rounded-transparent md-size'
+          : 'social-share-transparent'
+      } justify-content-end`}
+    >
+      <li>
+        <a
+          href={`https://www.facebook.com/sharer.php?u=${baseUrl}/${urlToShare}`}
+          target="_blank"
+        >
+          <FontAwesomeIcon icon={['fab', 'facebook-f']}/>
+        </a>
+      </li>
+      <li>
+        <a
+          href={`https://twitter.com/share?url=${baseUrl}/${urlToShare}&text=${encodeURI(
+            defaultText,
+          )}`}
+          data-related="@gweicz"
+          target="_blank"
+          data-show-count="false"
+        >
+          <FontAwesomeIcon icon={['fab', 'twitter']}/>
+        </a>
+      </li>
+      <li className={styles.copyLink}>
+        <a
+          href="#"
+          onClick={(event) => copyLink({ event, textToCopy: urlForClipboard })}
+        >
+          {isCopiedTooltip()}
+          <FontAwesomeIcon icon="link"/>
+        </a>
+      </li>
+    </ul>
+  )
+} else {
```

Comment:
Celá tahle sekce if-else se dá vyřešit jednodušeji. 

Necháš tam jenom ten obsah v té "else" větvi. A všechny ty "FontAwesomeIcon" komponenty takhle upravíš.
```javascript
<FontAwesomeIcon icon={['fab', 'facebook-f']} style={color ? {color: color} : {color: 'inherit'}}/>
```
Popřípadě to jde ještě takhle zjednodušit, ale to je pak už trochu matoucí.
```javascript
<FontAwesomeIcon icon={['fab', 'facebook-f']} style={color ? {color} : {color: 'inherit'}}/>
```